### PR TITLE
Upgraded to make it compatible with latest Spree versions

### PR DESCRIPTION
**Pull request by cescox**

### CODE REVIEW

File: spree_html_email.gemspec [hunk 1]
```diff
@@ -16,8 +16,8 @@ Gem::Specification.new do |s|
   s.require_path = 'lib'
   s.requirements << 'none'
 
-  s.add_dependency 'spree_core', '~> 1.0.0'
-  s.add_dependency 'premailer-rails3', '~> 1.1.0'
+  s.add_dependency 'spree_core'
+  # s.add_dependency 'premailer-rails'
```
Comment (joshnuss) [inline spree_html_email.gemspec:20]: @francesco-carrella why the move of premailer-rails from gemspec to Gemfile?

File: app/views/spree/shared/email/_order_details.html.erb [hunk 2]
```diff
@@ -91,22 +91,22 @@
   </tbody>
 </table>
 
-<% if Rails.application.railties.all.map(&:railtie_name).include? 'spree_digital' %>
-<% digital_items = @order.line_items.select{ |i| i.digital? } %>
-<% unless digital_items.empty? %>
-<div class="spacer_15">&nbsp;</div>
-<h2><span><%= t(:digital_download_links) %></span></h2>
-<p><%= t(:digital_email_description) %></p>
+<% if Rails.application.railties.map(&:railtie_name).include? 'spree_digital' %>
```
Comment (iloveitaly) [inline app/views/spree/shared/email/_order_details.html.erb:94]: Use [gem_available?](https://github.com/spree/spree/blob/7eb9caac7025d88371a2266c4cab99fe930263bf/core/app/helpers/spree/base_helper.rb#L127) here instead

File: app/views/spree/shared/email/_order_details.html.erb [hunk 3]
```diff
@@ -91,22 +91,22 @@
   </tbody>
 </table>
 
-<% if Rails.application.railties.all.map(&:railtie_name).include? 'spree_digital' %>
```
Comment (iloveitaly) [inline app/views/spree/shared/email/_order_details.html.erb:94]: I don't see the need to flatten the code structure. I like indented tabs in ERB/HTML – makes it easier to ready + parse through visually.

**Comment by joshnuss** [comment]
@francesco-carrella Thanks for the submission, imho the dependency on spree_digital is something that cant be accepted because all users may not want it. Sorry!

Happy to accept a pull request without it

**Pull request closed without merging**